cmd/devp2p/internal/ethtest/chain.go

```diff
@@ -68,6 +84,43 @@ func (c *Chain) Head() *types.Block {
 	return c.blocks[c.Len()-1]
 }
 
+func (c *Chain) GetHeaders(req GetBlockHeaders) (BlockHeaders, error) {
+	if req.Amount < 1 {
+		return nil, fmt.Errorf("no block headers requested")
+	}
+
+	headers := make(BlockHeaders, req.Amount)
```

Comment:
Might want to place an upper cap on `Amount`

Reply:
Just spoke with Felix about this and he thinks that it's okay to panic in a circumstance where the request amount is too large for the test to manually handle. We can put an explicit panic here too. What do you think?

Reply:
Well, I @fjl thinks so, then ok

Reply:
I think it's OK because panic here will just fail the test. If we try to handle it gracefully here, we won't notice if a client requests too much.